brave/brave-core
# Removes foreign keys

Resolves https://github.com/brave/brave-browser/issues/7682

## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (unit, browser, security tests) on

## Test Plan:
- install version 1.0 and populate data (visit sited, do tips, set recurring tips, etc)
- update to this version
- make sure that all data is still there

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on.
